### PR TITLE
BXMSPROD-990 drools-wb-scenario-simulation-editor-kogito-webapp-common artifact added

### PR DESCRIPTION
**Pull request by Ginxo**
**Thank you for submitting this pull request**

**JIRA**: https://issues.redhat.com/browse/BXMSPROD-990

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</pre>

### CODE REVIEW

**Comment by Ginxo** [comment]
let's for GA `Build Chain / Maven Build (8) (pull_request)` execution but locally works

**Comment by yesamer** [comment]
@Ginxo @mareknovotny Most probably this change can fix the issue, but I think add a `kogito` dependency in drools-webapp should be avoided.  drools-webapp doesn't need any kogito module, so we should try to understand why during drools-webapp compilation that module is compiled.

**Comment by danielezonca** [comment]
I agree with @yesamer , these dependencies should not be needed to compile. @Ginxo are you able to reproduce the error locally?

**Comment by Ginxo** [comment]
> I agree with @yesamer , these dependencies should not be needed to compile. @Ginxo are you able to reproduce the error locally?

Yes I can.
I agree with you guys, can we merge while we find the real reason?

File: drools-wb-webapp/pom.xml [hunk 1]
```diff
@@ -922,6 +922,24 @@
       <artifactId>drools-scenario-simulation-api</artifactId>
     </dependency>
 
+    <dependency>
```
Comment (yesamer) [inline drools-wb-webapp/pom.xml:925]: @Ginxo Can you please move the dependencies on block which starts in line 1096? Here the are others kogito dependencies which are temporary (probably for the same reason you need to add these new ones?) and planned to be removed with https://issues.jboss.org/browse/KOGITO-459 ? My suggestion is to link all these kogito dependencies in other to be managed (and removed) togheter.

**Review by yesamer — COMMENTED** [review]
@Ginxo Considering this issue blocks the builds and we haven't any clue of the reason, fine for me to proceed. However, I underline this should be considered a temporary solution, in the same way we did for other kogito dependencies described here. I understand the difficulties to find the root cause, I tried to analyze it but I didn't have any clue at the moment, so please continue your analysis after the merge, I can support you if you find a clue.

File: drools-wb-webapp/pom.xml (hunk 1 above)
Comment (yesamer) [reply drools-wb-webapp/pom.xml:925]: @Ginxo Or just put the same comment in this place, your choice :)

Comment (Ginxo) [reply drools-wb-webapp/pom.xml]: @yesamer moved, thanks

**Review by yesamer — APPROVED** [review]
Thank you @Ginxo.
 In my opinion the reason we need to add these new dependencies and the reason we added `kie-wb-common-stunner-kogito-api` (et others) it's the same.

**Review by danielezonca — APPROVED** [review]
Approving because it is blocking CI setup

**Review by mareknovotny — APPROVED** [review]
approving and @Ginxo will create a new jira for followup to fix this "unnecessary" linkage of kogito modules into Business Central components

**Pull request merged**